src/__tests__/__snapshots__/countryContainer.test.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
- [ ] Please, consider fixing the not passed test below:
<img width="556" alt="Screen Shot 2022-05-27 at 10 31 10" src="https://user-images.githubusercontent.com/49064106/170643810-05cba43a-6ff9-4126-9d81-0d345ff7323f.png">

Reply:
Thank you for your nice feedback now test are passing.
<img width="556" alt="screen shot" src="https://drive.google.com/file/d/1-LqXHe_Jlv-8gs0LlDdFjDnLag0Sh1RX/view?usp=sharing">

Reply:
![Screenshot from 2022-05-27 16-29-18](https://user-images.githubusercontent.com/93380132/170716600-c5028d15-6c79-42e5-a8cf-6ce0a7f891ca.png)
Thank you for your feedback, I implemented requested changes.
Here is the test screenshot that is passing